VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.html

```diff
@@ -1,13 +1,46 @@
 <div class="container">
     <h2 class="govuk-heading-s white">{{ 'private-consultation-participants.participants' | translate }}</h2>
-    <div class="participant-grid">
-        <ng-container *ngFor="let participant of getPrivateConsultationParticipants(); trackBy: trackParticipant">
+    <div class="participant-grid">        
+        <ng-container *ngFor="let groupMember of getGroupMembers();">
+            <ng-container *ngIf="isJohConsultation() && geMemberConsultationParticipants(groupMember).length > 0">
+                <div  class="language-icon">
+                    <img  class="small-crest" src="/assets/images/UkGovCrestWhite.png"[attr.alt]="'private-consultation-participants.crest-icon-alt' | translate" />
+                    <br>
+                </div>
+                <div class="panel-member-group">
+                    <ng-container *ngFor="let participant of geMemberConsultationParticipants(groupMember); trackBy: trackParticipant">        
```

Comment:
Instead of nesting an ngFor in an ngFor I would either:
1. Create a new component that takes the participants
2. Use the approach I suggested yesterday which just uses the hearing types & last-child
We've discussed both of these options & IMO they're both better solutions to this problem.

Reply:
component created